CodeEditModules/Modules/WorkspaceClient/src/Model/FileItem.swift

```diff
@@ -257,7 +257,10 @@ public extension WorkspaceClient {
             var fileUrl = self.url
             while FileItem.fileManger.fileExists(atPath: fileUrl.path) {
                 let previousName = fileUrl.lastPathComponent
-                fileUrl = fileUrl.deletingLastPathComponent().appendingPathComponent("\(previousName) copy")
+                let filextension = fileUrl.pathExtension
+                let fileName = filextension.isEmpty ? previousName : previousName.replacingOccurrences(of: ".\(filextension)", with: "")
```

Comment:
Fix the SwiftLint warning here

> Line Length Violation: Line should be 120 characters or less: currently 136 characters (line_length)

Just break the line into two right after the `?`